Toshi/Source/TClass.h

```diff
@@ -1,32 +1,36 @@
 #pragma once
-#include "TObject.h"
 
 namespace Toshi
 {
+	typedef void(*t_CreateTObject)();
+	typedef void(*t_CreateTObjectAtPlace)();
+	typedef void(*t_RegisterScriptingAPI)();
+
 	class TClass
 	{
 	public:
-		char* m_pcClassName; //0
-		void* m_pTObjectCTOR; //0x4
-		void* m_pTObjectDTOR; //0x8
-		void* m_pFunc; //0xC
-		void* m_pFunc2; //0x10
+		const char* m_pcClassName; //0
+		t_CreateTObject m_pTObjectCTOR; //0x4
+		t_CreateTObjectAtPlace m_pTObjectCTORAtPlace; //0x8
+		t_RegisterScriptingAPI m_pRegisterScriptingAPI; //0xC
+		void* m_pFunc; //0x10
 
-		Toshi::TClass* m_parent; //0x14
-		Toshi::TClass* m_attached; //0x18
-		Toshi::TClass* m_tclass; //0x1C
-		int m_version; //0x20
```

Comment:
Jpog says it's a version. 1 short for Major Version 1 short for minor version. m_version is the full version then. Checked in de blob and jpog and methods GetMinorVersion and GetMajorVersion appear in both games.

Reply:
fixed that and pushed the changes